src/api/grainConfig.js

```diff
@@ -11,6 +11,7 @@ export type GrainConfig = {|
   +balancedPerWeek?: number,
   +recentPerWeek?: number,
   +recentWeeklyDecayRate?: number,
+  +immediateNumPeriodsLookback?: number,
```

Comment:
I don't want any more fields added here because we would deprecate it immediately (per #2600). Please remove any changes to this file from this PR.

Reply:
Done.  This default to 1 for now.  We can add customization in the config after #2600 as suggested.